### PR TITLE
Documentation for APIv4: GET /users/{user_id}/audits

### PR DESCRIPTION
**Pull request by saturninoabril**
Documentation for APIv4: GET /users/{user_id}/audits

Github issue: [Mattermost/platform #5411](https://github.com/mattermost/platform/issues/5411)

### CODE REVIEW

File: source/v4/users.yaml [hunk 1]
```diff
@@ -551,4 +551,33 @@
...
+
+  /users/{user_id}/audits:
+    get:
+      tags:
+        - users
+      summary: Get user's audits
+      description: |
+        Get a list of audit by providing the user GUID.
+        ##### Permissions
+        Must be logged in as the user being updated or have the `edit_other_users` permission.
```
Comment (jwilander) [inline source/v4/users.yaml]: I think there's a copy/paste error here, can you change this to "Must be logged in as the user or have the `edit_other_users` permission."

Comment (saturninoabril) [inline source/v4/users.yaml]: my bad ;) ... now corrected. thanks!

**Comment by jwilander** [comment]
Looks good but I think you have a commit from your other branch in here `APIv4: GET /teams/{team_id}/members` and I think that's going to duplicate some documenation. Seems to be in some of your other PRs as well. Can you clear that commit out?

**Comment by saturninoabril** [comment]
Deleted the unrelated commit. thanks!

**Pull request merged**